commons/src/main/java/io/hyscale/commons/constants/ValidationConstants.java

```diff
@@ -44,7 +52,7 @@
             + SERVICE_NAME_REGEX + "')";
 
     public static final String INVALID_NAMESPACE_MSG = "Namespace \"{}\" is invalid. It must consist of lower case alphanumeric characters or '-', "
-            + "its length should be between 2 and 30."
+            + "its length should be between " + NAMESPACE_LENGTH_MIN + " and " + NAMESPACE_LENGTH_MAX + "."
             + "(regex used for validation is '"
             + NAMESPACE_REGEX + "')";
```

Comment:
Too many concatenations , better use StringBuilder

Reply:
On using StringBuildewe get the error:
"The value for annotation attribute Pattern.regexp must be a constant expression" at compilation in pattern based validation.

Reply:
Better We will have to fix this error .